src/script/index.js

```
@@ -82,3 +82,23 @@ window.addEventListener('DOMContentLoaded', () => {
+var i;
+
+// Loop through all accordion elements
+for (i = 0; i < acc.length; i++) {
+    // Add a click event listener to each accordion
+    acc[i].addEventListener('click', function () {
+        this.classList.toggle('footer__accordian--active');
+        var panel = this.nextElementSibling;
+        if (panel.style.maxHeight) {
+            panel.style.maxHeight = null;
+        } else {
+            panel.style.maxHeight = panel.scrollHeight + 'px';
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)